Makefile

```diff
@@ -112,7 +112,7 @@ pytest-unit:
 
 pytest-integration:
 	@docker compose -f docker-compose.yml -f docker-compose.integration-tests.yml up -d $(IMAGE_NAME)
-	@$(RUN) pytest -x -m integration
+	@docker compose run -e SNOWFLAKE_FIDESCTL_PASSWORD --rm $(IMAGE_NAME) pytest -x -m integration
```

Comment:
@ThomasLaPiana  what do you think about having to reference a pass through env variable here?

Reply:
this seems fine to me, not sure how else we'd do it